Gemfile

```diff
@@ -80,3 +90,18 @@ end
 group(:linux, :bsd, :mac_os_x, :solaris) do
   gem "ruby-shadow"
 end
+
+# TODO delete this when we figure out how to include the pushy windows dependencies
+# correctly
+platforms :mswin, :mingw do
```

Comment:
The right thing to do here is to update pushy client to export a window's gemspec.  Will be done in a different PR